### DISCUSSION TITLE
Any plan to support Expo? (with config plugin)

### DISCUSSION THREAD

**Discussion by arnlen** [innoveit/react-native-ble-manager#1079] [Ideas]
Since last discussions in 2020 (see [this issue](https://github.com/innoveit/react-native-ble-manager/issues/612)), Expo now supports native code (see this doc from from Expo "[Add custom native code](https://docs.expo.dev/workflow/customizing/)"). For example, the [react-native-ble-plx](https://github.com/dotintent/react-native-ble-plx) module can be integrated into Expo through [this dedicated config plugin](https://github.com/expo/config-plugins/tree/main/packages/react-native-ble-plx).

❓ Is there any plan to add Expo support? 🙏 

Thanks for your great work anyway. 🤗

**Comment by AndreOneti** [comment]
You don't need aby plugin, just use an [Expo Development build](https://docs.expo.dev/develop/development-builds/introduction/).

In my project work fine.

**Comment by stefnnn** [comment]
@arnlen, you may want to check out https://github.com/matthewwarnes/react-native-ble-manager-plugin from @matthewwarnes, which seems to be what you are looking for. Haven't tried it out yet though.

**Comment by arnlen** [comment]
Thank you for your replies.
I'll give your suggestions a try! 🤗

**Reply by stefnnn** [reply]
Yes, I tried it out and it works nicely so far 👍
